tests/unit/utils/parsing_test.py

```diff
@@ -251,50 +264,109 @@ def test_parse_eyelink_version(tmp_path, metadata, expected_version, expected_mo
 
     assert metadata['version_number'] == expected_version
     assert metadata['model'] == expected_model
+    assert metadata['time'] == time
```

Comment:
I think it makes sense to test the time field separately. keep the separate test function for time (maybe rename it to `test_parse_eyelink_datetime()` and check for `metadata['datetime']`